classes/EntityList.php

```diff
@@ -55,7 +55,8 @@ function __construct($entity_type, $module) {
         $fields = $this->getFields();
 
         $this->cols = array_keys($fields);
-        $this->formUrl = ExternalModules::getUrl(REDCAP_ENTITY_PREFIX, 'manager/entity.php');
+        $this->externalModule = new ExternalModules();
+        $this->formUrl = $this->externalModule->getUrl(REDCAP_ENTITY_PREFIX, 'manager/entity.php');
```

Comment:
```suggestion
        $this->formUrl = $this->module->getUrl(REDCAP_ENTITY_PREFIX, 'manager/entity.php');
```
EntityList already has a `module` object available, no need to create another.  
https://github.com/ctsit/redcap_entity/blob/c8cfdff0b80a3ab61b926841090726872fdb98f0/classes/EntityList.php#L47

Reply:
Except my change breaks something!